mava/adders/reverb/episode.py

```diff
@@ -29,25 +29,24 @@
 import tree
 from acme import specs, types
 from acme.adders.reverb import utils
-from acme.utils import tree_utils
+from acme.adders.reverb.episode import EpisodeAdder, _PaddingFn
 
 from mava.adders.reverb import base
 from mava.adders.reverb import utils as mava_utils
+from mava.adders.reverb.base import ReverbParallelAdder
 
 
-class ParallelEpisodeAdder(base.ReverbParallelAdder):
+class ParallelEpisodeAdder(EpisodeAdder, ReverbParallelAdder):
     """An adder which adds sequences of fixed length."""
 
     def __init__(
         self,
         client: reverb.Client,
         max_sequence_length: int,
         delta_encoded: bool = False,
-        chunk_length: Optional[int] = None,
         priority_fns: Optional[base.PriorityFnMapping] = None,
-        pad_end_of_episode: bool = True,
-        break_end_of_episode: bool = True,
-        max_in_flight_items: Optional[int] = 25,
+        max_in_flight_items: int = 1,
```

Comment:
I see the previous default for this was 25, do we want to set it to 1 now?

Reply:
For consistency with acme's episode adder (https://github.com/deepmind/acme/blob/master/acme/adders/reverb/episode.py).